DynamicTablesPkg/Include/ArchCommonNameSpaceObjects.h

```diff
@@ -46,6 +47,15 @@ typedef enum ArchCommonObjectID {
   EArchCommonObjPccSubspaceType4Info,           ///< 23 - Pcc Subspace Type 4 Info
   EArchCommonObjPccSubspaceType5Info,           ///< 24 - Pcc Subspace Type 5 Info
   EArchCommonObjPsdInfo,                        ///< 25 - P-State Dependency (PSD) Info
+  EArchCommonObjFadtSciInterrupt,               ///< 26 - FADT SCI Interrupt information

```

Comment:
Should these be in the x86/x64 arch namespace? These are certainly not used by Arm architecture.